storagetransfer/quickstart.go

```diff
@@ -2,17 +2,15 @@ package storagetransfer
 
 // [START storagetransfer_quickstart]
 import (
+	storagetransfer "cloud.google.com/go/storagetransfer/apiv1"
 	"context"
 	"fmt"
-	"io"
-	"log"
-
-	storagetransfer "cloud.google.com/go/storagetransfer/apiv1"
 	storagetransferpb "google.golang.org/genproto/googleapis/storagetransfer/v1"
+	"io"
```

Comment:
Core language dependencies should come first; run goimports to fix (intellij can do this for you).